jib-gradle-plugin/README.md

```diff
@@ -1,4 +1,4 @@
-[![experimental](http://badges.github.io/stability-badges/dist/experimental.svg)](http://github.com/badges/stability-badges)
+![experimental](https://img.shields.io/badge/stability-experimental-darkorange.svg)
 [![Analytics](https://cloud-tools-for-java-metrics.appspot.com/UA-121724379-2/jib-gradle-plugin)](https://github.com/igrigorik/ga-beacon)
```

Comment:
Maybe we should move these analytics badges to the bottom of the page too (to match the main readme, and also because the analytics badge looks slightly different and it looks weird).

Reply:
:+1: